dashboard/app/views/home/_survey_diversity.haml

```diff
@@ -22,7 +12,7 @@
           %a#privacylink{href: CDO.code_org_url('/privacy'), target: "_blank"} privacy policy
 
     = form_for(SurveyResult.new, html: {id: 'survey', onsubmit: "return window.submitSurvey(true)"}) do |f|
-      .survey_left
+      .survey_questions
         %ol
           %li
             Of your students using Code.org's courses, how many are
```

Comment:
I feel like there should be a colon here, if you're in there making style updates before you merge.

Reply:
I will double check this with Amanda (along with the green, which I agree looks a bit suspect).